### PR TITLE
DBAL 3 release post

### PR DESCRIPTION
**Pull request by beberlei**
Is there a change needed in some other file to make this visible?

### CODE REVIEW

File: source/blog/2020-11-17-dbal-3.0.0.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+We have released a new major version of Doctrine DBAL, version 3.0,0. This new
+major version comes almost 10 years after DBAL 2.0 was released on December
+2010, then coupled into the ORM 2.0.
+
+Today Doctrine DBAL is released independent of the ORM, thanks to Composer and
+sees its new major version before the ORM.
+
+This release was made possible foremost by
+[Sergei](https://twitter.com/srgmrzv), our primary DBAL maintainer, who has
```
Comment (morozov) [inline source/blog/2020-11-17-dbal-3.0.0.md:16]: There's nothing wrong with just Sergei but I wouldn't mind being referred by the full name.

**Review by morozov — COMMENTED** [review]
Looks great to me! Both from the form and content points of view.

The only thing that I'd try to improve is to end on a more positive note. E.g. there are some more improvements that didn't make into this release like API modernization, strict scalar types, better error handling, etc.

File: source/blog/2020-11-17-dbal-3.0.0.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+---
+title: "New Major Release: Doctrine DBAL 3.0"
+authorName: Benjamin Eberlei
+authorEmail: kontakt@beberlei.de
+permalink: /2020/11/17/dbal-3.0.0.html
+---
+
+We have released a new major version of Doctrine DBAL, version 3.0,0. This new
```
Comment (greg0ire) [inline source/blog/2020-11-17-dbal-3.0.0.md:8]:
```suggestion
We have released a new major version of Doctrine DBAL, version 3.0.0. This new
```

File: source/blog/2020-11-17-dbal-3.0.0.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+$stmt->fetchAllAssociative();
+```
+
+Many more changes have been made on the public API and also for the internals,
+but these are the most visible ones.
+
+### Upgrading to DBAL 3 from 2
+
+DBAL 3 is a real new major release with significant changes to the public API. Depending
+on your codebase a migration could require non-trivial work. However we do not intent
```
Comment (greg0ire) [inline source/blog/2020-11-17-dbal-3.0.0.md:78]:
```suggestion
on your codebase a migration could require non-trivial work. However we do not intend
```

File: source/blog/2020-11-17-dbal-3.0.0.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+to keep you hanging with DBAL 2 and a future migration:
```
Comment (greg0ire) [inline source/blog/2020-11-17-dbal-3.0.0.md:79]: Not 100% about this one but:
```suggestion
to leave you hanging with DBAL 2 and a future migration:
```

File: source/blog/2020-11-17-dbal-3.0.0.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+   upcoming PHP 8.0, so that there is no rush for you to upgrade to DBAL 3.
+
+2. DBAL 2.12 already includes forwards compatible API changes for all the new APIs,
+   so that you can migrate your code step by step to the new APIs already.
+
+3. Deprecated methods in DBAL 2.12 are tagged with the `@deprecated` doc-block
+   and static analysis tools such as Psalm, PHPStan and Phan can already help
+   you detect using or calling this deprecated code in your application.
+
+4. We intend to release another version of DBAL 2 branch which includes
```
Comment (greg0ire) [inline source/blog/2020-11-17-dbal-3.0.0.md:91]:
```suggestion
4. We intend to release another version of DBAL 2 which includes
```

**Comment by beberlei** [comment]
@morozov I added an outlook section at the end, what do you think?

**Comment by morozov** [comment]
> I added an outlook section at the end, what do you think?

Awesome!

**Review by morozov — APPROVED** [review]

**Review by SenseException — APPROVED** [review]

**Comment by SenseException** [comment]
I'll need to check the build of the website on the remote server to see why it isn't showing the latest changes. At some time this build- and deploy-server should be replaced by e.g. GitHub Actions.

**Pull request merged**